packages/modules/cra-template-desktop-viewer/template/README.md

```diff
@@ -10,7 +10,17 @@ Prior to running the app, you will need to add OIDC client configuration to the
 ITWIN_VIEWER_CLIENT_ID="native-xxxxxxxx"
 ```
 
-- You should generate a [client](https://developer.bentley.com/register/) to get started. The client that you generate should be for a desktop app and use the Visualization apis. You can add the default redirect uri (http://localhost:3000/signin-callback).
+- You should generate a [client](https://developer.bentley.com/register/) to get started. The client that you generate should be for a desktop app and use the following list of apis. You can add the default redirect uri (http://localhost:3000/signin-callback).
+
+- Scopes expected by the viewer are the following:
+
+  - `imodelaccess:read` is part of Visualization.
```

Comment:
I can change it, but you will still have to find iModels, I can remove this level of detail if its too much.

Reply:
Changed, did you want the "Enable the following scopes": text too ? (I didnt change it because in this case its not a walkthrough, merely information on what is expected)